README.md

```diff
@@ -52,7 +52,7 @@ Docker Compose installation options.
 The main Docker repo URL, common between Debian and RHEL systems.
 
     docker_apt_release_channel: stable
-    docker_apt_arch: "{{ 'arm64' if ansible_architecture == 'aarch64' else 'amd64' }}"
+    docker_apt_arch: "{{ {'x86_64': 'amd64', 'armv7l': 'armhf', 'aarch64': 'arm64'}[ansible_architecture] }}"
```

Comment:
Woah, I've never seen this filter syntax in Ansible/Jinja - is it some form of ternary or inline switch?

Reply:
This is just a dictionary. [These Python objects](https://jinja.palletsprojects.com/en/latest/templates/#literals) are supported by Jinja.